### ISSUE TITLE
About box needs more licenses

### ISSUE THREAD

**Issue by agoode** [jcreedcmu/twelf-mac#20]
There are a few licenses for used code that need to be shown in the about box as well as what's in the current box.

**Comment by jcreedcmu** [comment]
Agree. I welcome specific suggestions for the text you think is appropriate if you want to. Can summarize what's in LICENSE otherwise.

**Comment by jcreedcmu** [comment]
Please check if the current About text is satisfactory and suggest/merge changes if not. I'm not particularly attached to the wording, and very receptive to suggestions.

**Comment by agoode** [comment]
I think it won't fit on a screen and needs a scrolling text box.

You're supposed to include the full text of each license with the binary (see about:license on firefox). But if it's going to be in the about box, this is going to be super long. Maybe it just needs to go into a separate text file included with any binary distribution we make.

**Comment by jcreedcmu** [comment]
The "does it fit on a screen" is a good thought generally. I do want to make sure this works on a compact mac like the SE/30.

I think separate text file makes sense for meeting the obligation of the license being included with the binary distribution. My main goal for the about box is just that it fit the expected "texture" of an about box from the era. To my gut sense, a collection of copyright statements and attribution makes sense; not so much the full text of BSD or GPL or whatever. But if you have interesting examples, I'd love to see 'em.

**Comment by jcreedcmu** [comment]
Here is how it looks B&W on 512x342, after I decreased the image size and the font size: 
![image](https://github.com/jcreedcmu/twelf-mac/assets/1500822/c56167c8-22d8-4460-9836-0b6502b389a0)

**Comment by jcreedcmu** [comment]
Is this issue resolved now, given the resolution of https://github.com/jcreedcmu/twelf-mac/issues/17 ?

**Comment by agoode** [comment]
Probably once we have a `LICENSE` file?

**Comment by jcreedcmu** [comment]
[LICENSE](https://github.com/jcreedcmu/twelf-mac/commits/main/LICENSE) has existed for a little while now.

**Comment by jcreedcmu** [comment]
If you want some automation of building a tarball that includes it alongside the binary, that would maybe be nice.
